Java 8 compatibility for snapshots

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'slack-android-ng'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.osacky.flank.gradle:fladle:0.15.1-SNAPSHOT.
     Required by:
         project :
      > No matching variant of com.osacky.flank.gradle:fladle:0.15.1-SNAPSHOT:20210405.222926-5 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally but:
          - Variant 'apiElements' capability com.osacky.flank.gradle:fladle:0.15.1-SNAPSHOT declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
          - Variant 'runtimeElements' capability com.osacky.flank.gradle:fladle:0.15.1-SNAPSHOT declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

--- a/fladle-plugin/build.gradle.kts
+++ b/fladle-plugin/build.gradle.kts
@@ -162,3 +162,14 @@ tasks.withType(ValidatePlugins::class.java).configureEach {
   failOnWarning.set(true)
   enableStricterValidation.set(true)
 }
+
+// Ensure Java 8 Compatibilty. See https://github.com/runningcode/fladle/issues/246
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile::class.java).configureEach {
+  kotlinOptions.jvmTarget = "1.8"
+}
+
+java {
+  toolchain {
+    languageVersion.set(JavaLanguageVersion.of(8))
+  }
+}

